readme.md

```diff
@@ -39,6 +39,10 @@ git pull --rebase origin master
 ./infrastructure/scripts/environment_bootstrap.sh
 ```
```

Comment:
If this looks good, better to remove the `environment_bootstrap.sh` script and instructions so as to not maintain 2 sets of bootstrap scripts? (Then again, not a huge burden as the `environment_bootstrap.sh` has not changed much :smile: )

Reply:
@sentree / @benkroop - should we remove `environment_bootstrap.sh` or keep it around?